client/app/hosting/translations/Messages_fr_FR.xml

```diff
@@ -1196,6 +1198,17 @@
    <translation id="hosting_order_upgrade_price_TTC" qtlid="146190">Prix total TTC :</translation>
    <translation id="hosting_order_upgrade_success" qtlid="181266">La commande numéro <a href='{0}' target='_blank'>{1}</a> a bien été prise en compte.</translation>
    <translation id="hosting_order_upgrade_error" qtlid="225153">Une erreur est survenue lors de la modification de votre offre d'hébergement mutualisé.</translation>
+   <translation id="hosting_order_upgrade_modal_header">Changer d'offre</translation>
+   <translation id="hosting_order_upgrade_step_header_offer">Offre</translation>
+   <translation id="hosting_order_upgrade_step_header_duration">Durée</translation>
+   <translation id="hosting_order_upgrade_step_header_start_time">Heure de la mise à jour</translation>
+   <translation id="hosting_order_upgrade_step_header_contracts">Contrats</translation>
+   <translation id="hosting_order_upgrade_start_time_label">Sélectionnez l'heure de la mise à jour</translation>
+   <translation id="hosting_order_upgrade_start_time_explain">L'heure que vous choisissez correspond à la prochaine fois que cette heure sera atteinte dans votre fuseau horaire. Si vous choisissez une heure qui est déjà passée, la mise à jour s'effectuera demain à cette heure.</translation>
+   <translation id="hosting_order_upgrade_start_time_info">Le changement d'offre Cloudweb entraine une coupure de service d'environ 15 minutes maximum après le début de l'opération. Vous pouvez donc choisir un horaire de nuit afin de limiter l'impact sur vos sites web.</translation>
```

Comment:
```suggestion
   <translation id="hosting_order_upgrade_start_time_info">Le changement d'offre Cloud Web entraine une coupure de service d'environ 15 minutes maximum après le début de l'opération. Vous pouvez donc choisir un horaire de nuit afin de limiter l'impact sur vos sites web.</translation>
```

Is that make sense?

Reply:
Content is in the process of reviewing those texts, we'll see what they come up with.